shared/src/main/scala/mouse/feither.scala

```diff
@@ -42,6 +42,12 @@ final class FEitherOps[F[_], L, R](private val felr: F[Either[L, R]]) extends An
   def getOrElseIn[A >: R](right: => A)(implicit F: Functor[F]): F[A] =
     F.map(felr)(_.fold(_ => right, identity))
 
+  def getOrRaiseF[E](e: => E)(implicit F: MonadError[F, _ >: E]): F[R] =
+    getOrElseF(F.raiseError(e))
+
+  def getOrRaiseMsgF(msg: => String)(implicit F: MonadThrow[F]): F[R] =
```

Comment:
I feel that method is quite specific to have it within the library. It's more likely user-specific code, which is quite ok for some applications.

Reply:
I see, I was not sure about it but I've decided to add it for two main reasons:
- There is already in place `toTryMsg` 
- There are only 2 possible scenarios, either having a custom error (throwable or not) or using a `RuntimeException` just to have a message. 

So, since actually, this solution is quite common I'd like to keep it but I see your point.

Reply:
Oh, sorry, I've missed that `toTy Msg` method. So if we have something like that, we may continue using this approach.